files/scripts/asic_status.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+"""
+    bootstrap-asic
+"""
+try:
+    import re
+    import syslog
+    import sys
+    from sonic_py_common import daemon_base
+    from swsscommon import swsscommon
+    from sonic_py_common import device_info
+except ImportError as e:
+    raise ImportError(str(e) + " - required module not found")
+
+#
+# Constants ====================================================================
+#
+CHASSIS_ASIC_INFO_TABLE = 'CHASSIS_ASIC_TABLE'
+SELECT_TIMEOUT_MSECS = 5000
+
+def main():
+    if len(sys.argv) != 2:
+        raise Exception('Pass valid asic-id as argument')
+
+    args_asic_id = sys.argv[1]
+
+    # Get num asics
+    num_asics = 0
+    num_asics = device_info.get_num_npus()
```

Comment:
Please use this https://github.com/Azure/sonic-buildimage/blob/078e0e0410e4cd95f2ed6829d9fc55e411be66bc/src/sonic-py-common/sonic_py_common/multi_asic.py#L93

Reply:
Done.